paimon-core/src/main/java/org/apache/paimon/table/RollbackHelper.java

```diff
@@ -33,6 +34,7 @@
 import java.io.UncheckedIOException;
 import java.util.ArrayList;
 import java.util.Collections;
+import java.util.HashSet;
 import java.util.List;
 import java.util.Set;
 import java.util.SortedMap;
```

Comment:
Not only Rollback, you need to consider orphan file clean too.

Reply:
Yes, I think this can be done in another pr ?

Reply:
If the workload is not significant, you can consider completing it here

Reply:
OK